src/tomczyk-kamil/Fsd.Kamil.Cs/Fsd.Kamil.Cs.Ex2-3/Fsd.Kamil.Cs.Ex2-3/Animals/Animal.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Fsd.Kamil.Cs.Ex2_3.Animals
+{
+    public abstract class Animal
+    {
+        public int Number { get; set; }
```

Comment:
raczej number nie powinien być częścią zwierzaka. To liczba porządkowa. Co gdybym poprosił o sortowanie alfabetyczne po imionach ? Zmieni się kolejność i number będzie zły